From 91fe2d5a60d7e5f9ec1ffe42fb77ad062ab49ef6 Mon Sep 17 00:00:00 2001
From: Youxin Su <suyouxin@sporelab.io>
Date: Wed, 30 Jan 2019 10:57:43 +1300
Subject: [PATCH] replace random to urandom

---
 programs/pkey/gen_key.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pal-platform/Middleware/mbedtls/mbedtls/programs/pkey/gen_key.c b/pal-platform/Middleware/mbedtls/mbedtls/programs/pkey/gen_key.c
index a7f5c90a6..b8e9d0d0d 100644
--- a/pal-platform/Middleware/mbedtls/mbedtls/programs/pkey/gen_key.c
+++ b/pal-platform/Middleware/mbedtls/mbedtls/programs/pkey/gen_key.c
@@ -61,7 +61,7 @@ int dev_random_entropy_poll( void *data, unsigned char *output,
 
     *olen = 0;
 
-    file = fopen( "/dev/random", "rb" );
+    file = fopen( "/dev/urandom", "rb" );
     if( file == NULL )
         return( MBEDTLS_ERR_ENTROPY_SOURCE_FAILED );
 
-- 
2.17.1

